### DISCUSSION TITLE
How to fix "Attempt to get back from bad stream  file: volumeMesh.xyz at line 1."?

### DISCUSSION THREAD

**Discussion by wgtfly** [mdolab/dafoam#731] [Bug]
When I want to generate a mesh using a new wing profile based on the NACA0012 case, I have a problem. I found that someone else had a similar problem before, but I don't know how they solved it. Can you help me take a look? This is my profile file and the error text.
[](url)
[lower.txt](https://github.com/user-attachments/files/18262334/lower.txt)
[upper.txt](https://github.com/user-attachments/files/18262336/upper.txt)
/bin/bash: /home/dafoamuser/dafoam/packages/miniconda3/lib/libtinfo.so.6: no version information available (required by /bin/bash)
Generating mesh..

--> FOAM FATAL IO ERROR: 
Attempt to get back from bad stream

file: volumeMesh.xyz at line 1.

    From function bool Foam::Istream::getBack(Foam::token&)
    in file db/IOstreams/IOstreams/Istream.C at line 56.

FOAM exiting

--> FOAM FATAL ERROR: 
Cannot find file "points" in directory "polyMesh" in times "0" down to constant

    From function virtual Foam::IOobject Foam::fileOperation::findInstance(const Foam::IOobject&, Foam::scalar, const Foam::word&) const
    in file global/fileOperations/fileOperation/fileOperation.C at line 879.

FOAM exiting

--> FOAM FATAL ERROR: 
Cannot find file "points" in directory "polyMesh" in times "0" down to constant

    From function virtual Foam::IOobject Foam::fileOperation::findInstance(const Foam::IOobject&, Foam::scalar, const Foam::word&) const
    in file global/fileOperations/fileOperation/fileOperation.C at line 879.

FOAM exiting

--> FOAM FATAL ERROR: 
Cannot find file "points" in directory "polyMesh" in times "constant" down to constant

    From function virtual Foam::IOobject Foam::fileOperation::findInstance(const Foam::IOobject&, Foam::scalar, const Foam::word&) const
    in file global/fileOperations/fileOperation/fileOperation.C at line 879.

FOAM exiting

Generating mesh.. Done!
dafoamuser@b8285d3f7e36:~/mount$ ./Allclean.sh 
/bin/bash: /home/dafoamuser/dafoam/packages/miniconda3/lib/libtinfo.so.6: no version information available (required by /bin/bash)
Delete everything and resume to the default setup (y/n)?y
Cleaning...
dafoamuser@b8285d3f7e36:~/mount$ ./preProcessing.sh |tee log.preprocessing
/bin/bash: /home/dafoamuser/dafoam/packages/miniconda3/lib/libtinfo.so.6: no version information available (required by /bin/bash)
Generating mesh..

--> FOAM FATAL IO ERROR: 
Attempt to get back from bad stream

file: volumeMesh.xyz at line 1.

    From function bool Foam::Istream::getBack(Foam::token&)
    in file db/IOstreams/IOstreams/Istream.C at line 56.

FOAM exiting

--> FOAM FATAL ERROR: 
Cannot find file "points" in directory "polyMesh" in times "0" down to constant

    From function virtual Foam::IOobject Foam::fileOperation::findInstance(const Foam::IOobject&, Foam::scalar, const Foam::word&) const
    in file global/fileOperations/fileOperation/fileOperation.C at line 879.

FOAM exiting

--> FOAM FATAL ERROR: 
Cannot find file "points" in directory "polyMesh" in times "0" down to constant

    From function virtual Foam::IOobject Foam::fileOperation::findInstance(const Foam::IOobject&, Foam::scalar, const Foam::word&) const
    in file global/fileOperations/fileOperation/fileOperation.C at line 879.

FOAM exiting

--> FOAM FATAL ERROR: 
Cannot find file "points" in directory "polyMesh" in times "constant" down to constant

    From function virtual Foam::IOobject Foam::fileOperation::findInstance(const Foam::IOobject&, Foam::scalar, const Foam::word&) const
    in file global/fileOperations/fileOperation/fileOperation.C at line 879.

FOAM exiting

Generating mesh.. Done!

**Comment by friedenhe** [comment]
Your profile is not quite right, you need to manually truncate the last few points to create a blunt trailing edge. Refer to the profiles from the NACA0012 tutorials.
